package.json

```diff
@@ -33,9 +37,18 @@
     "swagger-ui-express": "^4.1.4"
   },
   "devDependencies": {
+    "chai": "^4.2.0",
+    "chai-http": "^4.3.0",
+    "coveralls": "^3.1.0",
     "eslint": "^7.9.0",
     "eslint-config-airbnb-base": "^14.2.0",
-    "eslint-plugin-import": "^2.22.0"
+    "eslint-plugin-import": "^2.22.0",
```

Comment:
What's the use of this package?

Reply:
That package has nothing to deal with testing. It is  among the changes I accepted after rebasing. And I think that it will be dealing with preventing issues with misspelling of file paths.